__tests__/cron/updateHighlightedViews.ts

```diff
@@ -0,0 +1,10 @@
+import { crons } from '../../src/cron/index';
+import cron from '../../src/cron/updateHighlightedViews';
+
+describe('updateHighlightedViews cron', () => {
+  it('should be registered', () => {
```

Comment:
I like this. Registering the worker gets missed sometimes and this would ensure they got listed.

Reply:
Yep, all the recent crons/worker have this 👌